ci/jenkins/jobs/default.yaml

```diff
@@ -0,0 +1,6 @@
+- defaults:
```

Comment:
I think you can remove this file, those variables are already defined in jenkins.